src/mame/drivers/mpu4unsorted.cpp

```diff
@@ -1934,3 +1904,18 @@ GAME(199?, m4luckwbd, m4luckwb,   mod4oki,    mpu4,    mpu4unsorted_state, init_
 GAME(199?, m4luckwbe, m4luckwb,   mod4oki,    mpu4,    mpu4unsorted_state, init_m4default,  ROT0,   "<unknown>","Lucky Wild Boar (MPU4) (set 6)",GAME_FLAGS )
 GAME(199?, m4luckwbf, m4luckwb,   mod4oki,    mpu4,    mpu4unsorted_state, init_m4default,  ROT0,   "<unknown>","Lucky Wild Boar (MPU4) (set 7)",GAME_FLAGS )
 
+/*****************************************************************************************************************************************************************************
+*
+* Four More
+* - does not appear to be a Barcrest codebase, and game seems to be unprotected
+* - produces no sound even when given an OKI rom from a different game, or with mod4ym, or mod2
```

Comment:
This set does produce PSG sound if `update_ay(m_pia6)` is changed to `update_ay(m_pia5)` in `mpu4_state::pia_ic6_cb2_w`.

Reply:
I can't find anything broken by doing that, and there have been some other silly little issues uncovered while looking at this lot, so I've made that change for now